examples/government/singletask_learning_bench/subjective/testenv/llm_judgement.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from sedna.common.class_factory import ClassType, ClassFactory
+from core.common.log import LOGGER
+
+__all__ = ["llm_judgement"]
+
+def extract_comprehensive_score(input_str):
+    # extract overall points
+    match = re.search(r"'综合得分': (\d+)", input_str)
```

Comment:
In the re.search of llm_judgement.py, we need to replace Chinese characters with English.